justfile

```diff
@@ -190,6 +190,4 @@ ci-musl-prepare: ci-prepare
     exit 1
 
 version-up version:
-    git grep -l "^version = .* # MARK: Version" | xargs sed -i 's/version = "[0-9]\.[0-9]\.[0-9]" # MARK: Version/version = "{{version}}" # MARK: Version/g'
-    sed -i s/_[0-9]_[0-9]_[0-9]_/_{{ replace(version, '.', '_') }}_/g  docs/src/user/basic_setup.md
-    sed -i 's/[0-9]\.[0-9]\.[0-9]/{{version}}/g'  docs/src/user/basic_setup.md
```

Comment:
Why can we delete them?

Reply:
I took the script to implement it separately. There is something wrong with this writing, and there is a problem with the replacement of the specific version number involved in the version upgrade. Of course, the version does not support aarch64, and the next version will support it.

Reply:
Will `scripts/release_tag.sh` replace the version tags in Cargo.toml?

Reply:
> Will `scripts/release_tag.sh` replace the version tags in Cargo.toml?

Then maybe I need to adjust. This script automatically creates a PR after we publish it, and automatically modifies it after the PR is merged. But I know Cargo.toml needs to be modified before merging. Do I understand correctly?

Reply:
Ideal for the release flow, I think:

1. Trigger a CI to prepare for the release. Update `Cargo.toml`s, docs, and so on
2. We merged it
3. Automatically push a new tag
4. Release CI starts run by pushing a new tag.